bin/node/runtime/pangolin/src/lib.rs

```diff
@@ -921,10 +981,8 @@ impl OnRuntimeUpgrade for CustomOnRuntimeUpgrade {
 		// let _ = Ring::deposit_creating(&BridgeMillauMessages::relayer_fund_account_id(), 1 << 50);
 		// --->
 
-		darwinia_header_mmr::migration::initialize_new_mmr_state::<Runtime>(b"HeaderMMR", 50);
-		darwinia_relay_authorities::migration::migrate::<Runtime, EthereumRelayAuthoritiesInstance>(
-			b"Instance1DarwiniaRelayAuthorities",
-		);
+		darwinia_ethereum_relay::migration::migrate(PARCEL);
+		darwinia_relayer_game::migration::migrate::<Runtime, EthereumRelayerGameInstance>();
```

Comment:
Can we remove the commented line `<-- Hack for local test ... --->` in `on_runtime_upgrade()` and `pre_upgrade()` now?

Reply:
I think we can't. Until the parity-bridges-common fix that. Have you tried that?

Nodes will panic without those codes.